### PR TITLE
UsableComingIn addition to Runways

### PR DESCRIPTION
**Pull request by DIProgan**
Will do Maridia later

### CODE REVIEW

File: region/cleanup.json [hunk 1]
```diff
@@ -432,8 +443,12 @@
           "sparking": {
             "runways": [
               {
-                "length": 8,
+                "length": 6,
```
Comment (fdtaillefer) [inline region/cleanup.json]: I thought Homing Geemer Room was the section of West Ocean we split off from the rest of the room. That has a basic 12-tile runway with a door on each side. What place are you at?

**Comment by fdtaillefer** [comment]
Do you think you can add the property to runways in the schema? I think there's probably a way to explicitly state a default value of `true`.

File: region/cleanup.json (hunk 1 above)
Comment (DIProgan) [reply region/cleanup.json]: The room left of that was my belief.

File: schema/m3-region.schema.json [hunk 2]
```diff
@@ -215,6 +215,12 @@
                               "null"
                             ]
                           },
+                          "UsableComingIn": {
```
Comment (fdtaillefer) [inline schema/m3-region.schema.json]: If not sure if this thing is case sensitive. but to be consistent I suggest the name use camel case (so usableComingIn) and that the same capitalization be used in the "$id"

File: region/crateria/east.json [hunk 3]
```diff
@@ -565,7 +565,7 @@
           "sparking": {
             "runways": [
               {
-                "length": 12,
+                "length": 4,
```
Comment (fdtaillefer) [inline region/crateria/east.json]: There seems to be the same thing with water as on the right side. Does it need two runways as well?

File: schema/m3-region.schema.json (hunk 2 above)
Comment (DIProgan) [reply schema/m3-region.schema.json]: It's not case sensitive but aligning it as requested

**Pull request merged**